apps/ledger-live-mobile/src/index.tsx

```diff
@@ -158,6 +160,12 @@ function App() {
     [datadogFF?.params, automaticBugReportingEnabled],
   );
 
+  useEffect(() => {
+    if (typeof ldmkSolanaSignerFeatureFlag?.enabled === "boolean") {
+      setSolanaLdmkEnabled(ldmkSolanaSignerFeatureFlag?.enabled);
+    }
+  }, [ldmkSolanaSignerFeatureFlag]);
```

Comment:
We could update the deps to be a bit more precise but I don't really mind it like this
```suggestion
  }, [ldmkSolanaSignerFeatureFlag?.enabled]);
```